napari/layers/base/_base_layer_interface.py

```diff
@@ -0,0 +1,86 @@
+from abc import abstractmethod
+
+from napari.utils.base_interface import BaseInterface
+
+
+class BaseLayerInterface(BaseInterface):
+    """
+    Empty implementation of BaseInterface. Defines a set of methods shared between the data layer visual
+    rendering and controls.
+    """
+
+    @abstractmethod
+    def _on_refresh_change(self):
```

Comment:
Can we add docstrings to these? It'd be good for subclass writers to know what they are meant to do when defining these.